Zonos/amino
# Feat/inline icon

## Jira issue

[FE-657 - Docs: display inline icon styling](https://myzonos.atlassian.net/browse/FE-657)

## Description

This PR adds an optional `inlineBlock` prop to the icons and adjusts the icon creation script accordingly.

## Testing
Domain: https://icons.amino.zonos.com

1. Go to the Icons story
2. Inspect an Icon with the dev tools
3. Switch the `inlineBlock` prop toggle on and off
4. Observe the `display` css change

## Todo

- [ ] Bump version and add tag
